cmd/oteltestbedcol/builder-config.yaml

```diff
@@ -2,50 +2,50 @@ dist:
   module: github.com/open-telemetry/opentelemetry-collector-contrib/cmd/oteltestbedcol
   name: oteltestbedcol
   description: OpenTelemetry Collector binary for testbed only tests.
-  version: 0.76.1-dev
+  version: 0.76.2-dev
   otelcol_version: 0.76.1
```

Comment:
Does this need bumped?

Reply:
I believe that refers to the core collector version, which I did not need to bump.